Supporting Documents/BS_SD.adoc

```diff
@@ -891,57 +891,16 @@ This Section describes EAs for AVA_VAN.1 step by step following the order of AVA
 
 The evaluator shall search publicly available information that is published after the publication date of the <<Toolbox>> to look for new artefact species. New artefact species are those artefacts that are out of scope of the <<Toolbox>> and need to be made in a completely different way with significantly different materials that are not covered by the <<Toolbox>>.
```

Comment:
Noticed this while reading @n-kai's PR:

> New artefact species are those artefacts that are out of scope

This may need to be reworded, since "out of scope" also includes non-basic attack potential artefacts. Artefacts that are "significantly different" from or made using significantly different materials but still basic attack potential.

Reply:
I have updated the text to remove the out of scope phrase. I think this should work well.